src/main/resources/templates/db_scripts/note_data_update_metadata_user_info.sql

```diff
@@ -30,3 +34,9 @@ language 'plpgsql';
 DROP TRIGGER IF EXISTS set_note_data_md_json_trigger ON note_data CASCADE;
 
 CREATE TRIGGER set_note_data_md_json_trigger BEFORE UPDATE ON note_data   FOR EACH ROW EXECUTE PROCEDURE set_note_data_md_json();
+
+DROP TRIGGER IF EXISTS set_note_data_md_trigger ON note_data CASCADE;
```

Comment:
Jus a quick sanity check here -> we are dropping and creating "set_note_data_md_json_trigger". Is "set_note_data_md_trigger" another trigger defined elsewhere?

Reply:
@sduvvuri-ebsco `set_note_data_md_trigger`, as well as other triggers, are created by Postgres Client. In the logs, you can find the information about triggers/function creation.

Reply:
The sample code for trigger:
![Screen Shot 2019-05-06 at 2 41 38 PM](https://user-images.githubusercontent.com/37537790/57223189-ecb9dc00-700d-11e9-8635-f57141c310a5.png)